src/dotnet-svcutil/lib/src/CommandProcessorOptions.cs

```diff
@@ -1025,6 +1025,6 @@ public string ToTelemetryString()
                 .Where(o => o.CanSerialize)
                 .Select(o => $"{o.Name}:[{OptionValueParser.GetTelemetryValue(o)}]")
                 .Aggregate((num, s) => num + ", " + s).ToString();
-        }
+        }       
```

Comment:
nit, can you please remove the extra spaces?

Reply:
Done